FreeRTOS/FreeRTOS
# Update Backward Compatibility Flag

Description
-----------
Add backward compatibility with IPv4 single endpoint branch by adding following changes:

- Update the backward compatibility flag for IPv4 only build to ipconfigIPv4_BACKWARD_COMPATIBLE.
- Rename FreeRTOS_GetUDPPayloadBuffer_ByIPType to FreeRTOS_GetUDPPayloadBuffer_Multi.
- Rename FreeRTOS_IPStart to FreeRTOS_IPInit_Multi

Test Steps
-----------
Run Demos on Jenkins.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
